doc/nrf/ug_nrf91_features.rst

```diff
@@ -160,6 +160,213 @@ The :ref:`nrfxlib:nrf_modem` is released as an OS-independent binary library in
+The orbital data can be received faster using A-GPS.
+
+Due to the clock bias on the receiver, there are four unknowns when looking for a GPS fix - latitude, longitude, altitude, and clock bias.
+This results in solving an equation system with four unknowns, and therefore a minimum of four satellites must be tracked to acquire a fix.
+
+Enhancements to GNSS
+====================
+
+When GNSS has not been in use for a while or if the device is in relatively weak signaling conditions, it might take longer to acquire a fix.
+To improve this, Nordic Semiconductor has implemented the following methods for acquiring a fix in a shorter time:
+
+•	A-GPS or P-GPS or a combination of both
+•	Low accuracy mode
+
+Assisted GPS (A-GPS)
+---------------------
+
+A-GPS is commonly used to improve the Time to first fix (TTFF) by utilizing a connection (for example, over cellular) to the internet to retrieve the Almanac and Ephemeris data.
+A connection to an internet server that has the Almanac and Ephemeris data is several times quicker than using the slow 50 bps data link to the GPS satellites.
+There are many options to retrieve this A-GPS data.
+Two such options are using `nRF Cloud`_ and SUPL.
+|NCS| provides example implementations for both these options.
+The A-GPS solution available through nRF Cloud has been optimized for embedded devices to reduce protocol overhead and data usage.
+This, in turn, results in the download of reduced amount of data, thereby reducing data transfer time, power consumption, and data costs.
+See :ref:`nrfxlib:gnss_int_agps_data` for more information about the retrieval of A-GPS data.
+
+Predicted GPS (P-GPS)
+---------------------
+
+P-GPS is a form of assistance, where the device can download up to two weeks of predicted satellite Ephemerides data.
+Normally, devices connect to the cellular network approximately every two hours for up-to-date satellite Ephemeris information or they download the Ephemeris data from the acquired satellites.
+P-GPS enables devices to determine the exact orbital location of the satellite without connecting to the network every two hours with a trade-off of reduced accuracy of the calculated position over time.
+Note that P-GPS requires more memory compared to regular A-GPS.
+
+Also, note that due to satellite clock inaccuracies, not all functional satellites will have Ephemerides data valid for two weeks in the downloaded PGPS package.
+This means that the number of satellites having valid predicted Ephemerides reduces in number roughly after ten days.
+Hence, the GNSS module needs to download the Ephemeris data from the satellite broadcast if no predicted Ephemeris is found for that satellite to be able to use the satellite.
+
+nRF Cloud A-GPS compared with SUPL library
+----------------------------------------------
+
+The SUPL library uses approximately 229 kB of flash memory and 68 kB of RAM, which is +50% of flash and +37% of RAM usage compared to nRF Cloud A-GPS library.
```

Comment:
These numbers can't be for the SUPL library alone. These numbers are probably for nRF Cloud library (without nRF Cloud A-GPS) + SUPL library vs. nRF Cloud library with nRF Cloud A-GPS.

When modem_shell (which does not have any nRF Cloud related stuff) is compiled with and without SUPL support the delta is 100 kB flash and 22 kB SRAM. This is the memory consumed by the SUPL library and the associated application implementation needed.

Reply:
Correct, these numbers are from A-GPS sample using SUPL vs. nrf cloud A-GPS memory usage.
I will change this to make it clearer.